lib/glimesh/events_team.ex

```diff
@@ -111,7 +111,7 @@ defmodule Glimesh.EventsTeam do
   def delete_event(eventid) do
     %Event{id: String.to_integer(eventid)}
     |> Event.changeset()
-    |> Repo.delete()
+    |> Repo.delete(eventid)
```

Comment:
I don't think this change does what you expect, the documents say this param is for options, not IDs.

Reply:
I was trying to fix the delete button because it wasn't working, when I tested it did delete the event now.